config/config.go

```diff
@@ -48,9 +50,16 @@ type IPFSConfig struct {
 }
 
 type APIConfig struct {
-	ListenAddr   string `mapstructure:"listen-addr"`
-	WriteTimeout int64  `mapstructure:"write-timeout"`
-	ReadTimeout  int64  `mapstructure:"read-timeout"`
+	Enabled               bool   `mapstructure:"enabled"`
+	ListenAddr            string `mapstructure:"listen-addr"`
+	GrpcConnectionTimeout int64  `mapstructure:"grpc-connection-timeout"`
+	WriteTimeout          int64  `mapstructure:"write-timeout"`
+	ReadTimeout           int64  `mapstructure:"read-timeout"`
+}
+
+type GRPCConfig struct {
+	ListenAddr        string `mapstructure:"listen-addr"`
+	ConnectionTimeout int64  `mapstructure:"connection-timeout"`
```

Comment:
```suggestion
	Enabled               bool   `mapstructure:"enabled"`
	ListenAddr            string `mapstructure:"listen-addr"`
	GrpcConnectTimeout int64  `mapstructure:"grpc-connect-timeout"`
	WriteTimeout          int64  `mapstructure:"write-timeout"`
	ReadTimeout           int64  `mapstructure:"read-timeout"`
}

type GRPCConfig struct {
	ListenAddr        string `mapstructure:"listen-addr"`
	ConnectTimeout int64  `mapstructure:"connect-timeout"`
```
I'd suggest to rename these parameters since these are timeout configurations for the [TCP CONNECT](https://pkg.go.dev/net#DialTimeout) operation.

Reply:
Ah, it would be correct to rename only `APIConfig.GrpcConnectionTimeout` to `APIConfig.GrpcConnectTimeout` since grpc-gateway is a grpc client that tries to establish a connection with a grpc server.

Unlikely, It makes sense to have `GRPCConfig.ConnectionTimeout`.

Reply:
OK. I fixed it(https://github.com/medibloc/panacea-oracle/pull/79/commits/3978966a3cb9259333dc7738d4de898e02c87463).
Thank you